contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -2689,6 +2699,169 @@ pltsql_detect_numeric_overflow(int weight, int dscale, int first_block, int nume
 	return (total_digit_count > TDS_NUMERIC_MAX_PRECISION);
 }
 
+/*
+ * Throw a common error message while casting to time datatype
+ */
+static void gen_func_time_in_error()
```

Comment:
may be add a macro, rather than function

Reply:
Sure will do that.